### PR TITLE
Make extension event handler public

### PR DESCRIPTION
**Pull request by jrmoulton**
Makes it possible to create signals from async tasks

Didn't look like this would be problematic to me

### CODE REVIEW

**Comment by Zoxc** [comment]
A function which returns it probably makes more sense.

File: src/ext_event.rs [hunk 1]
```diff
@@ -34,6 +34,10 @@ impl ExtEventHandler {
     }
 }
 
+pub fn add_ext_trigger(trigger: Trigger) {
```
Comment (dzhou121) [inline src/ext_event.rs:37]: I'm wondering if it's better to let the function to create the `Trigger` and return it?

Comment (jrmoulton) [reply src/ext_event.rs:37]: Probably about the same to me. But actually now that I'm thinking again I would take the trigger as an argument but call the function register_ext_trigger. 

Either works for me

Comment (dzhou121) [reply src/ext_event.rs:37]: `register_ext_trigger` does look better

**Pull request merged**